dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/cli/commands.py

```diff
@@ -90,3 +91,16 @@ def config(getter, setter, init) -> None:
         for key, val in setter:
             set_single_config(key, val)
         click.echo("Set configuration done.")
+
+
+@cli.command()
+@click.option(
+    "--yaml_file",
```

Comment:
maybe `--yaml-file` is better for long, and `-f` is for short

Reply:
done